### DISCUSSION TITLE
[FEATURE REQUEST] Reduce visual prominence of the real-time sync indicator

### DISCUSSION THREAD

**Discussion by Private-Fragrance** [standardnotes/forum#2672] [Suggestions]
There's a checkmark above my notes that shows real-time syncing. This is good. (I realize there's a way to turn off the sync-indicator but I _like_ the feature ...)

On dark mode especially, _the check mark is distracting_. It's very contrasting against the editor background. And being animated real-time, it has a flickering effect. 

**Suggested fix:** Keep the iconography of a check mark, but make it monochromatic and decrease contrast vs. the app background. For example, the "pin" icon is usable yet does not demand attention. 

In summary: Make the syncing checkmark the same visual style as the other icons in that row so that it is less distracting. Keep it animated, just make it less noticeable. 

Thanks for considering this.
